warehouse/core/src/main/java/datawave/util/threads/Threads.java

```diff
@@ -0,0 +1,9 @@
+package datawave.util.threads;
+
+import java.lang.Thread.UncaughtExceptionHandler;
+
+public class Threads {
```

Comment:
I would ditch this Threads class and just use `new DatawaveUncaughtExceptionHandler()` in BulkInputFormat. We don't need this wrapper.

Reply:
Sounds good! Wasn't sure if we should include it in case it's used down the line, but that's probably unnecessary future proofing.